src/blocks/Icons/Icons.tsx

```diff
@@ -16,7 +16,7 @@ const Icons = ({title, size = 's', items}: IconsBlockProps) => {
         <div className={b({size})}>
             {title && <BlockHeader className={b('header')} title={title} colSizes={{all: 12}} />}
             {items.map((item) => (
-                <a
+                <a // should we render a Link if it is in LocationContext?
```

Comment:
I think yes, we can render Link everywhere, but it may not support properties like `ref`, 'rel' and `onClick`, so it can be a problem in some cases you marked below